### PR TITLE
Add simple bundletool installation

### PR DESCRIPTION
**Pull request by Heezay**
By default we have bundletool_version const in android.rs file that can be changed

### CODE REVIEW

File: crossbundle/cli/src/commands/install/sdkmanager.rs [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    pub fn default_file_path(&self) -> crate::error::Result<PathBuf> {
+        let default_file_path = dirs::home_dir()
+            .ok_or_else(|| crate::error::Error::PathNotFound(PathBuf::from("$HOME")))?
+            .join(self.sdk_file_name());
+        Ok(default_file_path)
+    }
+
+    /// Set sdk root for sdkmanager storing
+    pub fn set_sdk_root(&self) -> crate::error::Result<PathBuf> {
+        let root = Path::new("AppData")
```
Comment (enfipy) [inline crossbundle/cli/src/commands/install/sdkmanager.rs:57]: This should be `windows` only.

File: crossbundle/cli/src/commands/install/mod.rs [hunk 2]
```diff
@@ -1,20 +1,48 @@
...
+/// Download jar file and save it in directory
+pub fn create_jar_file(
+    download_url: String,
+    jar_path: std::path::PathBuf,
+) -> crate::error::Result<()> {
+    let response = ureq::get(&download_url)
+        .call()
+        .map_err(crate::error::Error::DownloadFailed)?;
+
+    let mut out = std::fs::File::create(&jar_path).map_err(|cause| {
```
Comment (enfipy) [inline crossbundle/cli/src/commands/install/mod.rs:35]: I think we can write to created file right away with [write](https://doc.rust-lang.org/stable/std/io/trait.Write.html#tymethod.write).

Comment (Heezay) [reply crossbundle/cli/src/commands/install/mod.rs]: I think this is not so simple as it seems. Response struct doesn't implement std::io::write trait

Comment (Heezay) [reply crossbundle/cli/src/commands/install/mod.rs]: We have returned Response struct when ureq::get() function was invocated.

**Review by enfipy — APPROVED** [review]

**Pull request merged**